### PR TITLE
Updated Algorithm Caesar's Cipher

### PR DESCRIPTION
**Pull request by atjonathan**
Refer #1094

### CODE REVIEW

File: Algorithm-Caesars-Cipher.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
 - The one used here is ROT13 where the value of the letter is shifted by 13 places. e.g. 'A' ↔ 'N', 'T' ↔ 'G'.
 - You have to shift it back 13 positions, such that 'N' ↔ 'A'.
 
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Caesars-Cipher.md]: Remember to add links....

File: Algorithm-Caesars-Cipher.md [hunk 2]
```diff
@@ -64,7 +70,13 @@ function rot13(str) {
 - The for loop is used to loop through each character of the input string.
 - If the character is not uppercase English alphabets(i.e. its ascii doesn't lie between 65 and 91 ), we'll leave it as it is and [continue](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue) with next iteration.
 - If it's the uppercase English alphabet, we'll subtract 13 from it's ascii code.
-- If the ascii code is less than 78, it'll get out of range when subtracted by 13 so we'll add 26 (number of letters in English alphabets) to it so that after A it'll go back to Z. e.g. M(77) ↔ 77-13 = 64(Not an English alphabet) +26 = 90 ↔ Z(90)
+- If the ascii code is less than 78, it'll get out of range when subtracted by 13 so we'll add 26 (number of letters in English alphabets) to it so that after A it'll go back to Z. e.g. M(77) ↔ 77-13 = 64(Not an English alphabet) +26 = 90 ↔ Z(90).
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Caesars-Cipher.md]: Remember to add links....

File: Algorithm-Caesars-Cipher.md [hunk 3]
```diff
@@ -94,11 +106,20 @@ function rot13(str) {
...
 ```
 
+### Code Explanation:
+
+NEEDED
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Caesars-Cipher.md]: Remember to add links....

File: Algorithm-Caesars-Cipher.md [hunk 4]
```diff
@@ -111,10 +132,16 @@ function rot13(str) { // LBH QVQ VG!
...
-> - The arrow function applies the `rot13` transform on every uppercase letter from English alphabet present in the given string.
+- `String.prototype.replace` [function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) lets you transform a `String` based on some pattern match (defined by a regular expression), and the [transformation function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter) (which is applied to each of the pattern matches).
+- [Arrow function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) syntax is used to write the function parameter to `replace()`.
+- `L` represents a single unit, from every pattern match with `/[A-Z]/g` - which is every uppercase letter in the alphabet, from `A` to `Z`, present in the string.
+- The arrow function applies the `rot13` transform on every uppercase letter from English alphabet present in the given string.
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Caesars-Cipher.md]: Remember to add links....

**Comment by Rafase282** [comment]
# general links

https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-CharCodeAt
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/String.fromCharCode

# Solution 1

https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Map
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Split
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Join

# Solution 2
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Regex-Resources
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test

# Solution 3
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Replace

Still needs an explanation for solution 2, I suggest you just explain with your own words what the code does. That's what i do.

**Comment by Rafase282** [comment]
https://github.com/atjonathan/wiki/pull/5

File: Algorithm-Caesars-Cipher.md [hunk 5]
```diff
@@ -94,11 +105,20 @@ function rot13(str) {
...
 }
 
-
 // Change the inputs below to test
 rot13("LBH QVQ VG!");
 ```
 
+### Code Explanation:
+
+NEEDED
```
Comment (atjonathan) [inline Algorithm-Caesars-Cipher.md]: Just need to fix this...

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**